compiler/testData/checkLocalVariablesTable/copyFunction.kt

```diff
@@ -1,6 +0,0 @@
-data class someClass(val a: Double, val b: Double)
```

Comment:
Why this test is removed (and others)?

Reply:
copyFunction was removed because it was already ported yet not removed (localVariables/copyFunction.kt) Same goes for destructuringInLambdas (though I have some corner cases for that coming up).